io/src/test/scala/fs2/io/file/FileSpec.scala

```diff
@@ -63,9 +63,14 @@ class FileSpec extends BaseFileSpec {
         .resource(blockingExecutionContext)
         .flatMap { bec =>
           tempFile
-            .flatTap(modifyLater)
-            .flatMap(path => file.keepReading[IO](path, bec, 4096, 250.millis))
+            .flatMap { path =>
+              file
+                .keepReading[IO](path, bec, 4096, 25.millis)
```

Comment:
Do this instead: `file.keepReading(...).concurrently(modifyLater(path))` -- then you don't need the `Left`/`Right` mapping or the `collect`.

Reply:
Great! I didn't know about `concurrently`...

By the way, is there any documentation, apart from in the source code, where all these methods are explained?